tasks/prime.py

```diff
@@ -350,6 +356,7 @@ def create_mto_shipment(self, overrides=None):
         if overrides:
             overrides_local.update(overrides)
         payload = self.fake_request("/mto-shipments", "post", PRIME_API_KEY, overrides=overrides_local)
+        guarantee_unique_agent_type(payload["agents"])
```

Comment:
I think I've seen it done elsewhere but it might not be obvious that this modifies the payload variable directly instead of returning a new dict.